### ISSUE TITLE
React Native support

### ISSUE THREAD

**Issue by viesti** [portkey-cloud/aws-clj-sdk#19]
[React Native](http://facebook.github.io/react-native/) sits on top of JavaScriptCore (at least last time I checked). Would be good to check that #1  aligns with usage of `aws-clj-sdk` in React Native projects

**Comment by dupuchba** [comment]
really love the idea, we might wait after finishing the 5 protocols right ? How do you see it ?

**Comment by viesti** [comment]
AWS has a project called [Amplify](https://github.com/aws/aws-amplify) which has React components for authentication with Cognito and probably a set of helpers for making requests to AWS API's, but probably also to API's made with API Gateway intended to work as a backend for the mobile app. I haven't used or looked at Amplify, or the current state of ClojureScript with React Native, but I'd think that we have at least possibility of providing a Clojure native experience.

I'd think that it's best to finish service support, although RN app uses probably quite limited set of services (say DynamoDB and S3). Spinning up few servers with EC2 API isn't necessary, although might be fun, for some definition of fun :)

**Comment by dupuchba** [comment]
I've used a yeard ago cljs & react-native. If it compiles to cljs I don't see why `aws-clj-sdk` could not work

**Comment by cgrand** [comment]
I guess it’s about host api. You don’t have the same apis (network, crypto ) in React Native and in Node.

**Comment by viesti** [comment]
Yes, now that my mind refreshes, RN uses [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) for HTTP. Don't know what's the thing used in Node, maybe some XMLHttpRequest wrapper? But yeah, a survey of the host api is in order.
